pxtsim/simdriver.ts

```diff
@@ -130,6 +131,7 @@ namespace pxsim {
             frame.setAttribute('sandbox', 'allow-same-origin allow-scripts');
             frame.sandbox.value = "allow-scripts allow-same-origin"
             let simUrl = this.options.simUrl || ((window as any).pxtConfig || {}).simUrl || "/sim/simulator.html"
+            if (light) simUrl+= '?light=1';
```

Comment:
This breaks offline caching. Pass light in the simulator options

Reply:
I do do that, but issue is simulator options are only passed in once loaded. Which means by that time, the SVGs have been constructed. I needed a way to pass in that option on initialization. 
Thoughts?

Reply:
You'll need to encode that info in the hashtag arguments

Reply:
Cool yeah that should be easy. thanks